app/Scripts.hs

```diff
@@ -58,6 +58,7 @@ graphScripts = do
          "/static/js/common/utilities/util.js",
          "/static/js/common/image_conversion.js",
          "/static/js/common/graph_image.js",
+         "/static/js/vendor/bootstrap.min.3.1.1.js",
```

Comment:
So now that we're getting into styling details: I'd prefer not to use bootstrap here, if the only purpose is to recreate the css for the modal. It really should be possible to just use the existing modal CSS.